website/docs/r/sql_database_instance.html.markdown

```diff
@@ -228,6 +228,10 @@ The `settings` block supports:
     for more details and supported versions. Postgres supports only shared-core machine types,
     and custom machine types such as `db-custom-2-13312`. See the [Custom Machine Type Documentation](https://cloud.google.com/compute/docs/instances/creating-instance-with-custom-machine-type#create) to learn about specifying custom machine types.
 
+The optional `settings.advanced_machine_features` subblock supports:
```

Comment:
https://github.com/hashicorp/terraform-provider-google/issues/14813

This gets formatted so that all options below this appear to be within the advanced_machine_features subblock

https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_database_instance#threads_per_core

Putting them above this subblock should fix the issue